fv3core/fv3core/testing/translate_dyncore.py

```diff
@@ -125,6 +125,7 @@ def __init__(
         # TODO: Fix edge_interpolate4 in d2a2c_vect to match closer and the
         # variables here should as well.
         self.max_error = 2e-6
+        self.ignore_near_zero_errors = {"wsd": True}
```

Comment:
Is there a reason this is not passed from the override file?

Reply:
Maybe @mcgibbon would know?

Reply:
The units of ws (m/s) make it such that a 1e-18 difference is indeed very small, and shouldn't affect validation for any backend. My understanding is the override file is for backend-specific settings, not just as the default place to put validation constraints (though honestly, I'd feel better if all validation constraints were in yaml files and none were on translate classes like this).